src/index.ts

```diff
@@ -81,6 +108,9 @@ export class Piwik {
 
   private getUrl(): string {
     if (typeof window !== "undefined") {
+      if(document.location.pathname === "React App"){
```

Comment:
Does React Native set `document.location.pathname` to `"React App"`?

Reply:
regular react for some reason was O.o atleast on my local hahaha! i can double check this

Reply:
Regular react? Do you mean `react-dom`, `react-ssr`, or `react-native`?

Reply:
hm mm maybe i got it wrong.. i thought it was giving that but... im a take this out